### PR TITLE
Update changelog for 1.3.0 release

### PR DESCRIPTION
**Pull request by pschorf**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,12 +4,21 @@ All notable changes to this project will be documented in this file
...
+- Added disable-mea-culpa-retries flag, from @pschorf
+- Added docker and minimesos support, from @dpo
+- Added support for job groups in simulator, from @mforsyth
+- Added /failure_reasons API endpoint, from @mforsyth
+- Added expected-runtime to job description, from @dpo
+- Added /settings API endpoint, from @dpo
 ### Changed
 - Require an explicit reason when changing shares or quotas (from @mforsyth).  This intentionally breaks backwards compatibility.
 - Optimized matching code to speed schedule time @wyegelwel
+- Require a reason for changing shares/quota, from @mforsyth
```
Comment (wyegelwel) [inline CHANGELOG.md]: This is already mentioned on line 18

Comment (pschorf) [inline CHANGELOG.md]: Removed, thanks.

**Comment by wyegelwel** [comment]
Should the version be 1.3.0? The changes are more than bug fixes

**Comment by pschorf** [comment]
Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,12 +4,20 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
  
-## [Unreleased]
+## [1.3.0] - 2017-05-05
 ### Added
 - /unscheduled_jobs API endpoint, from @mforsyth
+- Added application to job description, from @dpo
+- Added disable-mea-culpa-retries flag, from @pschorf
+- Added docker and minimesos support, from @dpo
```
Comment (dposada) [inline CHANGELOG.md]: Maybe just "Added Dockerfile", since that's what is relevant to consumers?

Comment (pschorf) [inline CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,12 +4,20 @@ All notable changes to this project will be documented in this file
...
+## [1.3.0] - 2017-05-05
 ### Added
 - /unscheduled_jobs API endpoint, from @mforsyth
+- Added application to job description, from @dpo
+- Added disable-mea-culpa-retries flag, from @pschorf
+- Added docker, from @dpo
+- Added support for job groups in simulator, from @mforsyth
+- Added /failure_reasons API endpoint, from @mforsyth
+- Added expected-runtime to job description, from @dpo
+- Added /settings API endpoint, from @dpo
```
Comment (dposada) [inline CHANGELOG.md]: Nitpick: @dposada instead of @dpo

Comment (pschorf) [inline CHANGELOG.md]: Done

**Comment by wyegelwel** [comment]
Can you add a line about optimizing ranking as well?

File: CHANGELOG.md (hunk 3 above)
Comment (dpo) [inline CHANGELOG.md]: Thank you! :-)

**Comment by pschorf** [comment]
Done

**Pull request merged**